### PR TITLE
Amazon-AWS-describe-load-balancers

### PR DESCRIPTION
**Pull request by mihaitusa**
Op && unit test.
Signed off by Tusa Mihai: tusa.mihai@gmail.com

### CODE REVIEW

**Comment by Bonczidai** [comment]
@tethryus @victorursan WDYT about these `Changes Unknown when pulling 54908f4 on Amazon-AWS-describe-load-balancers into ** on master**.` kinf of messages by Coveralls? Is this the desired way? `Unknown` and stuff?

**Comment by mihaitusa** [comment]
C'mon @victorursan ... goodies are waiting... 😄

File: cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/loadbalancing/DescribeLoadBalancers.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+     * http://docs.aws.amazon.com/elasticloadbalancing/latest/APIReference/API_DescribeLoadBalancerAttributes.html
+     *
+     * @param endpoint          Optional - Endpoint to which request will be sent.
+     *                          Default: "https://elasticloadbalancing.amazonaws.com"
+     * @param identity          ID of the secret access key associated with your Amazon AWS or IAM account.
+     *                          Example: "AKIAIOSFODNN7EXAMPLE"
+     * @param credential        Secret access key associated with your Amazon AWS or IAM account.
+     *                          Example: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+     * @param proxyHost         Optional - proxy server used to connect to Amazon API. If empty no proxy will be used.
+     *                          Default: ""
```
Comment (victorursan) [inline cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/loadbalancing/DescribeLoadBalancers.java]: empty defaults are not needed ... they are redundant and in some cases they are not set at all if "isEmpty" ...  same goes for all inputs with empty default

Comment (mihaitusa) [inline cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/loadbalancing/DescribeLoadBalancers.java]: @victorursan  What should I do with those inputs?...

Comment (victorursan) [inline cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/loadbalancing/DescribeLoadBalancers.java]: just delete ' Default: "" '

Comment (mihaitusa) [inline cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/loadbalancing/DescribeLoadBalancers.java]: @victorursan Done.

**Comment by mihaitusa** [comment]
@Bonczidai ... pls merge it when you have some spare time!

**Comment by victorursan** [comment]
@mihaitusa I'll merge it if it's ok with you :)

**Pull request merged**